modules_in_project/finam/transaqConnector.js

```diff
@@ -16,36 +16,10 @@ const arrayUnikStringCloseOpenPositions = []; // массив уникальны
 // относительный путь в виндовс не всегда работает корректно, иногда существующий файл не находится
 // #endregion
 
-const isTransaqConnected = {
-    Hft: false,
-    NoHft: false,
-};
 
 // #region параметры подключения
 const board = 'FUT';
 
-// #region объект с логинами, паролями, dll файлами hft и НЕ hft, объектами dll
-const objectAccountsAndDll = {
```

Comment:
it must be global object for multiusers

Reply:
yes I have corrected that in the new push